core/src/trezor/enums/__init__.py

```diff
@@ -574,6 +582,24 @@ class MessageType(IntEnum):
         SolanaAddress = 903
         SolanaSignTx = 904
         SolanaTxSignature = 905
+        ThpCreateNewSession = 1000
+        ThpPairingRequest = 1006
```

Comment:
Should we mark the unused values (e.g. 1001-1005 here, also below) as deprecated?

Reply:
Added `reserved` keyword in 20135989753b3d1b9dbde2da7e827eec74ef0495 since `MessageType` generally seems to have no intra-block holes in the ID allocation. These were never used in a released version so we can use them in the future.